unified-prototype/unified-plugin/plugin-android/src/main/java/org/gradle/api/experimental/android/application/AndroidApplicationDependencies.java

```diff
@@ -16,12 +16,9 @@
 
 package org.gradle.api.experimental.android.application;
 
-import org.gradle.api.artifacts.dsl.DependencyCollector;
 import org.gradle.api.experimental.common.ApplicationDependencies;
 import org.gradle.declarative.dsl.model.annotations.Restricted;
 
-@SuppressWarnings("UnstableApiUsage")
 @Restricted
 public interface AndroidApplicationDependencies extends ApplicationDependencies {
```

Comment:
Do we even need `AndroidApplicationDependencies` anymore? Or can we just use `ApplicationDependencies` in its place now?

Reply:
Probably not.  Might have to get rid of this if nothing else makes its way into these.